sound/soc/sof/control.c

```diff
@@ -89,24 +89,26 @@ int snd_sof_volume_put(struct snd_kcontrol *kcontrol,
 	struct snd_sof_dev *sdev = scontrol->sdev;
 	struct sof_ipc_ctrl_data *cdata = scontrol->control_data;
 	unsigned int i, channels = scontrol->num_channels;
+	int change = 0;
```

Comment:
not sure the change should be also u32 type?

Reply:
I've kept that the same as the return type. Not sure about u32, but I see bool is also used:  https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/soc-io.c#L112

Reply:
@dragosht the way you compute the change suggests the use of a 'bool' indeed.

Reply:
@plbossart makes sense - bool it is.